### PR TITLE
WDPD-141 Create PaymentMethod folder

### PR DESCRIPTION
**Pull request by ps-amehic**
### This PR

* Moves payment methods to PaymentMethod folder
* Removes boolean parameter from getName function

### Jira Ticket
https://jira.parkside.at/browse/WDPD-141

### CODE REVIEW

File: Controller/Admin/Transaction/TransactionTabPostProcessing.php [hunk 1]
```diff
@@ -19,6 +19,7 @@
 use Wirecard\Oxid\Core\PostProcessingHelper;
 use Wirecard\Oxid\Core\TransactionHandler;
 use Wirecard\Oxid\Model\Transaction;
+use Wirecard\Oxid\Model\PaymentMethod\PaymentMethod;
```
Comment (ps-holler) [inline Controller/Admin/Transaction/TransactionTabPostProcessing.php]: Why was this added? I don't think it's used.

Comment (ps-amehic) [reply Controller/Admin/Transaction/TransactionTabPostProcessing.php]: Removed 👍
